release-calendar/src/client/components/ReleaseCard.tsx

```diff
@@ -0,0 +1,184 @@
+          .title
+      }`;
+    } else if (search == Channel.PERCENT_EXPERIMENTAL) {
+      return `${
+        channelTitles.find(
+          (channel) => channel.channel == Channel.PERCENT_EXPERIMENTAL,
+        ).title
+      }, ${
+        channelTitles.find((channel) => channel.channel == Channel.PERCENT_BETA)
+          .title
+      }`;
+    } else {
+      return channelTitles.find((channel) => channel.channel == search).title;
+    }
+  }
+
+  render(): JSX.Element {
+    return (
+      <div className={'event-card'}>
+        <div className={this.props.eventApi.classNames[0]}>
+          <div className={'event-top'}></div>
+          <div className={'event-content'}>
+            <div className='content-row'>
+              <div className={'big-emoji'}>
+                <span role='img' aria-label={'train'}>
+                  {'🚄'}
+                </span>
+              </div>
+              <h2
+                className={
+                  'title-row'
+                }>{`Release ${this.props.eventApi.title}`}</h2>
+            </div>
+            <div className='content-content'>
+              {/* TODO: uncomment below when cherrypick is connected 
+              (will be adding cherrypick attribute to releaseDates) */}
+              {/* {this.state.releaseDates.cherrypick && (
+                <div className={'content-row'}>
+                  <div className={'emoji'}>
+                    <span role='img' aria-label={'flower'}>
+                      {'🌸'}
+                    </span>
+                  </div>
+                  <div className={'text'}>{'Cherrypick!'}</div>
+                </div>
+              )} */}
+              <div className={'content-row'}>
+                <div className={'emoji'}>
+                  <span role='img' aria-label={'laptop'}>
+                    {'🔗'}
+                  </span>
+                </div>
+                <div className={'text'}>
+                  <a
+                    href={`https://github.com/ampproject/amphtml/releases/tag/${this.props.eventApi.title}`}
+                    target='_blank'
+                    rel='noopener noreferrer'>
+                    {'GitHub Release'}
+                  </a>
+                </div>
+              </div>
+              <h3 className={'subtitle-row'}>{'Release History'}</h3>
+              {this.state.releaseDates != null &&
+                this.state.releaseDates.dates.map((row) => {
+                  const match = this.history.find(
+                    (type) => type.channel == row.channel,
+                  );
+                  return (
+                    match && (
+                      <div className={'content-row'} key={match.emoji}>
+                        <div className={'emoji'}>
+                          <span role='img' aria-label={'row.emojiName'}>
+                            {match.emoji}
+                          </span>
+                        </div>
+                        <div className={'text'}>
```

Comment:
Do you need the curly brackets or can it be `className='text'`? Same question for the rest of this PR.

Reply:
No, I don't. I'll get clean it up